package.json

```diff
@@ -81,7 +81,8 @@
                         "bat": "",
                         "shellscript": "bash",
                         "fsharp": "fsi",
-                        "csharp": "scriptcs"
+                        "csharp": "scriptcs",
+                        "vbscript": "cscript //Nologo"
```

Comment:
From the VS Code [vb definition](https://github.com/Microsoft/vscode/blob/master/extensions/vb/package.json), the language id of `.vbs` file is `vb` not `vbscript`. How do you set it as `vbscript` and make it work?